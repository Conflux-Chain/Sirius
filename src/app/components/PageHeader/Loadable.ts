/**
 *
 * Asynchronously loads the component for PageHeader
 *
 */

import { lazyLoad } from 'utils/loadable';

export const Footer = lazyLoad(
  () => import('./index'),
  module => module.default,
);
