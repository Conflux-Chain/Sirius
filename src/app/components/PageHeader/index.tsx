/**
 *
 * Footer
 *
 */
import React, { memo } from 'react';
import styled from 'styled-components/macro';
import { media } from './../../../styles/media';

const Wrapper = styled.div`
  font-size: 1.7143rem;
  font-weight: 500;
  color: #1a1a1a;
  line-height: 2.2857rem;
  margin-bottom: 1.1429rem;

  ${media.s} {
    font-size: 1.2857rem;
    font-weight: 500;
    color: #050505;
    line-height: 1.8571rem;
  }
`;

const PageHeader = memo(({ children }) => {
  return <Wrapper>{children}</Wrapper>;
});

export default PageHeader;
