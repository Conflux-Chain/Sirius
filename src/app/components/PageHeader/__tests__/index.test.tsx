import React from 'react';
import { render } from '@testing-library/react';

import PageHeader from '..';

describe('<PageHeader  />', () => {
  it('should match snapshot', () => {
    const pageHeader = render(<PageHeader>PageHeader</PageHeader>);
    expect(pageHeader).toMatchSnapshot();
  });
});
