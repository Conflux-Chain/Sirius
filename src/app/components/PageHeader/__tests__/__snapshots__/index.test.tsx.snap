// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageHeader  /> should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c0 {
  font-size: 1.7143rem;
  font-weight: 500;
  color: #0f1327;
  line-height: 2.2857rem;
  margin-bottom: 1.1429rem;
}

@media (max-width:600px) {
  .c0 {
    font-size: 1.2857rem;
    font-weight: 500;
    color: #050505;
    line-height: 1.8571rem;
  }
}

<body>
    <div>
      <div
        class="c0"
      >
        PageHeader
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="PageHeader__Wrapper-se2wca-0 bAanhF"
    >
      PageHeader
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
